app/views/proposals/_form.html.haml

```diff
@@ -1,11 +1,17 @@
 %fieldset.margin-top
   %legend.fieldset-legend Event(s)
-  %p
-    Which conference(s) does your proposal apply to? Select at least one conference.
-  .form-group
-    = f.select :internal_occurrences,
-      options_for_select(Proposal.allOccurrences, proposal.object.occurrences),
-      {}, {class: 'multiselect proposal-occcurrencees', multiple: true}
+  - error_class = proposal.errors.include?(:internal_occurrences) ? 'field_with_errors' : ''
+  .form-group(class=error_class)
+    %label.control-label.required(for='proposal_internal_occurrences')
+      Which conference(s) do you want to apply to?
+      %span(title='required')
+        *
+    %div
+      = f.select :internal_occurrences,
+        options_for_select(Proposal.allOccurrences, proposal.object.occurrences),
+        {}, {class: 'multiselect proposal-occurrences', multiple: true}
+      - if proposal.errors.include?(:internal_occurrences)
+        %span.help-block can't be blank
```

Comment:
That part is just a hack. I wish I could get the HTML generated by RoR form helpers but I couldn’t achieve that so I manually wrote the HTML.